entities.csv

```diff
@@ -1,3 +1,5 @@
+5-HT

```

Comment:
As a convention, we've used underscores for FamPlex entries so following that logic, this would be `5_HT`.